testdata/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains x509 certificates used in cloud-to-prod interop tests.
+For tests within gRPC-Go repo, please use the files in testsdata/x509
```

Comment:
Optional: if you want to enforce this, you can add a grep in vet.sh, to make sure nobody other than `interop/` uses those files.